lib/galaxy_test/api/test_datatypes.py

```diff
@@ -84,6 +84,20 @@ def test_converter_present_after_toolbox_reload(self):
         self._assert_status_code_is(response, 200)
         assert converters == len(response.json())
 
+    def test_edamp_formats(self):
```

Comment:
```suggestion
    def test_edam_formats(self):
```
I assume that's a typo ?

Reply:
Absolutely :) thanks for the catch